test-data/configuration_tests/input/jobs/snake_oil_simulator.py

```diff
@@ -184,6 +184,6 @@ def roundedInt(value):
 
     ecl_sum.fwrite()
 
-    with open("time_map.txt", "w") as f:
+    with open("time_map.txt", "w", encoding="utf-8") as f:
         for t in time_map:
```

Comment:
Changing variable name t -> time_step?
And is {t}==%s ?

Reply:
Fixed. `t` already has type `str`.